src/module/script.js

```diff
@@ -0,0 +1,46 @@
+import { displayTask } from './index.js';
+
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+const checkbox = document.querySelector('#myTasksList, #checkbox');
+
+const completedTasks = () => {
+  const uncompletedTodos = tasks.filter((element) => element.completed !== true);
+  tasks = uncompletedTodos.map((element, index) => {
+    element.index = index + 1;
+    return element;
+  });
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  // loadTasks(newTask);
+};
+
+const checkCheckbox = ({ index, status }) => {
+  tasks[index - 1].completed = status;
+  displayTask();
+  completedTasks();
+};
```

Comment:
- [ ] As suggested by the previous reviewer kindly make sure the users can update the status of their `todo` tasks by `toggling` the checkbox. Currently, when the checkbox is toggled you get an error which doesn't create a good user experience. Please fix this issue so your users can have a good experience using your app. See below for the error details:
<img width="725" alt="Screenshot 2023-05-13 at 12 37 02" src="https://github.com/Bhekisile/To-do-list/assets/107192076/bcf4e815-9b6f-41f7-bafe-e6a8ffa2665c">


- [ ] Also your `add` function doesn't work as expected. As suggested by the previous reviewer please make sure the fix this issue because currently when a user tries to `add` a task and they click on the `add` `icon` beside the `input` field they get the below error. 
<img width="725" alt="Screenshot 2023-05-13 at 12 37 34" src="https://github.com/Bhekisile/To-do-list/assets/107192076/df8aa6dd-b55b-454b-88ae-df26173545dc">
But when they hit `enter` from their `keyboard` the task is being added and stored in the `localStrage` but not visible on the `UI`. Fixing this issue will help create a good user experience.